code/modules/mob/living/silicon/robot/dogborg/dog_sleeper_vr.dm

```diff
@@ -388,7 +388,7 @@
 				to_chat(hound, "<span class='notice'>Your stomach is currently too full of fluids to secrete more fluids of this kind.</span>")
 			else if(patient.reagents.get_reagent_amount(chem) + 10 <= 20) //No overdoses for you
 				patient.reagents.add_reagent(chem, inject_amount)
-				drain(750) //-750 charge per injection
+				drain(50) //-50 charge per injection
```

Comment:
This isn't actually quite right. The 50 charge on the borg hypo is actually removing 'charges' from the matter_synthesizer, which I started following exactly how it gets power from the cell but then lost it because it's late. I'll look at it tomorrow.